src/encrypt.js

```diff
@@ -92,7 +92,7 @@ function encrypt(message, subscription) {
  * {@link https://tools.ietf.org/html/draft-ietf-httpbis-encryption-encoding-00}
  * @param  {Buffer} clientPublicKey The client's public key
  * @param  {Buffer} serverPublicKey Our public key
- * @return {Buffer}
+ * @return {Buffer} context
```

Comment:
Does ESLint complain if you don't have a description here? I find that in many cases documenting the return value is redundant because it says in the function description what will be returned.

I accept that this is a personal style thing, but I don't think that adding the word 'context' here adds to the readability of the code. If Google's chosen rules say otherwise, though, then I'll go with that.